frontend/src/views/Pages/UserProfile/UserCard_Language.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div
+    id="change_language"
+    class="bg-transparent pt-3 pb-3"
+    style="background-color: #ebebeba3 !important"
+  >
+    <b-container>
+      <div>
+        <b-row class="mb-4 text-right">
+          <b-col class="text-right">
+            <a @click="!editLanguage ? (editLanguage = !editLanguage) : cancelEdit()">
+              <span class="pointer mr-3">{{ $t('form.changeLanguage') }}</span>
+              <b-icon v-if="!editPassword" class="pointer ml-3" icon="pencil" />
```

Comment:
```suggestion
              <b-icon v-if="!editLanguage" class="pointer ml-3" icon="pencil" />
```
Is this correct?

Reply:
no  the name is in all  show...[Password, Language, ...]